plio/views.py

```diff
@@ -223,13 +239,35 @@ def download_data(self, request, uuid):
         # schema name to query in
         schema_name = OrganizationTenantMiddleware().get_schema(self.request)
 
+        if BIGQUERY["enabled"]:
+            gcp_service_account_file = "/tmp/gcp-service-account.json"
```

Comment:
should we keep it as part of the repo as in `plio-analytics` and move it to `.gitignore`? Also, will need to update deployment / installation steps to include info about this file.